### ISSUE TITLE
Dead tabs when system wakes up from sleep

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#9865]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Dead tabs when system wakes up from sleep

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.17.13
rev	f7242c7
Muon	4.1.7

- Steps to reproduce:
    1. Install 0.17.3 build
    2. Use browser normally and ensure there is enough browsing history and multiple tabs opened with backgrounded tabs
    3. With browser window active put computer to sleep -> Start -> Power button -> Sleep
    4. Wake up system from sleep and try to switch to tabs, none are useable
    5. Quit and restart browser, able to switch to all tabs
    
- Actual result:
Dead tabs when system wakes up from sleep

- Expected result:
Tabs should work normally and should be able to switch when system wakes up from sleep

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Mostly yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:
cc: @bbondy

**Comment by bridiver** [comment]
@srirambv is this a new issue or can you repro in previous releases?

**Comment by srirambv** [comment]
@bridiver Couldn't reproduce the same on 0.16.9.

**Comment by bridiver** [comment]
Thanks @srirambv. Does it go back to earlier 0.17 versions? If we know the exact version it was introduced in it will be easier to track down the problem

**Comment by bbondy** [comment]
@srirambv

**Comment by srirambv** [comment]
@bridiver I wasn't able to consistently reproduce this on all the previous 0.17.x versions. It did happen once on 0.17.9 during which I found #9972.

**Comment by bridiver** [comment]
@srirambv couldn't reproduce consistently or couldn't reproduce at all?

**Comment by srirambv** [comment]
@bridiver  couldn't reproduce on all versions except 0.17.9. Out of 3 attempts only once it happened on 0.17.9. 

+1 from community user(Numpty) https://community.brave.com/t/brave-freezes-completely-after-a-while-of-non-usage/3444/

**Comment by bridiver** [comment]
0.17.9 went from muon 4.1.3 to 4.1.4, but this was the only change https://github.com/brave/muon/commit/2befbb73253124c71ec0fd8e9b25f9e6bccb165f
I'll try on master and see if I can repro

**Comment by zentagonist** [comment]
Punting back to Sriram - can you still repro this issue?

**Comment by srirambv** [comment]
@zentagonist  I haven't seen this happen on the 0.18.x build. Have tried on both new profile and existing profile but haven't seen dead tabs.

**Comment by ricardosaracino** [comment]
i just got back home opened up from sleep

closed brave and process list looks like this
![image](https://user-images.githubusercontent.com/759253/30231815-bc922d3c-94ba-11e7-87c4-8f7da80247b4.png)

**Comment by srirambv** [comment]
Just reproed this on 0.19.29. Dead tabs after system woke up from sleep. Unable to switch to pinned tabs or other tabs. Unable to quit or close had to kill via task manager

cc: @bridiver

**Comment by mattnewport** [comment]
I see this pretty consistently on 0.19.95 but it doesn't seem to affect all tabs. 'Dead' tabs cannot be selected, tabbed to or closed.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/tabs-freeze-all-the-time/11568 on 0.19.105

**Comment by vitorschwartz** [comment]
Dear Brave community, 
Any estimate of when this will be fixed? IMO, this is one of the most annoying bugs of Brave =/
I'm back using Chrome because I get these zombie tabs so often with Brave, and I was wasting too much time to reopen the browser a couple times per day...
Thank you very much!

**Comment by jonathansampson** [comment]
+1 from Twitter: https://twitter.com/christianbute/status/943897324237008898

**Comment by jonathansampson** [comment]
I just received another +1 from Twitter over DM.

**Comment by AwesomeTech10** [comment]
I think this issue got fixed

**Comment by NejcZdovc** [comment]
@petemill is this one still open?

**Comment by petemill** [comment]
@NejcZdovc Technically not, since we disabled the feature that's causing the issue, but if we re-enable the feature the problem is still there, and I'm working on it in #13120

**Comment by NejcZdovc** [comment]
@petemill can you please add closing keyword for this issue into #13120?

**Comment by petemill** [comment]
I'm actually closing this as it's covered pretty well in #10673

**Comment by srirambv** [comment]
@petemill shoudl #10673 and this be marked as `QA/no-qa-needed`  based on https://github.com/brave/browser-laptop/issues/10673#issuecomment-367114377

**Comment by LaurenWags** [comment]
+1 from community DM
